MANIFEST.in

```diff
@@ -0,0 +1 @@
+include prometheus_ricoh_printer_exporter/printers.csv
```

Comment:
I would prefer that printers are passed in via a `-c CONFIG` option, like the Raritan and Eaton exporters do.

This does not need to block this PR, but should be done before we deploy this.